src/javascript/_common/base/gtm.js

```diff
@@ -162,10 +162,55 @@ const GTM = (() => {
         pushDataLayer(gtm_data);
     };
 
+    const pushTransactionData = (response, extra_data = {}) => {
+        if (!isGtmApplicable() || ClientBase.get('is_virtual')) return;
```

Comment:
@morteza-binary What do you suggest we do to not use the `ClientBase` and use the store value instead?

Reply:
why? it's already imported here, is there any harm?

Reply:
Since the change of no-reloading client switch, it will create inconsistency with our source of truth, which is`client_store`. The value should be handled by that object, see: 

https://github.com/binary-com/binary-static/blob/master/src/javascript/app_2/Stores/client_store.js#L120-L123

Reply:
@easteregg you're right we will face some issues in the future when we want to split the new_ui and `binary_static` and keep the data of `client_store` like other stores by `base_store` but unfortunately, this file is used directly from both projects.
We can make a card to refactor it in the future.